### DISCUSSION TITLE
blog idea: how to check your website/page for social media tags

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn.com#271] [Ideas]
@nandhinidevie, @gsalunke please post all URLs etc here as comments so it can be included in the future blog post.

**Comment by gsalunke** [comment]
How do I inspect my social post, before posting it on social platforms? 
If I am using my blog URL, and want to share my article, is opengraph image showing up? 

If you have above questions in your mind, here is the solution.
Each social platform provided their tools to validate your posts. I have found few, and they can be searched easily via using google search.

Post validators:
1. LinkedIn: https://www.linkedin.com/post-inspector/inspect/
2. Twitter: https://www.bannerbear.com/tools/twitter-card-preview-tool/
3. Discord, FaceBook and Linkedin: https://www.opengraph.xyz/
